standards/node_standards.md

```diff
@@ -1,7 +1,7 @@
 ## Draft Node.js Standards
```

Comment:
It's not part of your changes, but would suggest its included; remove the word `Draft` from the Javascript and Node.js standards.

Reply:
Good spot, will definitely do it.